### ISSUE TITLE
Incorrect vaccination rates nearly 0% for some locations

### ISSUE THREAD

**Issue by jackc271** [microCOVID/microCOVID#1275]
**Describe the bug**
Vaccination rate has some data bug for some locations such as San Francisco and Santa Clara reporting nearly 0% vaccination rate.

Under location details, I see Percent of population fully vaccinated: 0% for San Francisco, 1% for Santa Clara. Most locations seem fine though.

Result: "Their vaccine" yes/no options give incorrect estimates.

**Link to microCOVID scenario**
https://www.microcovid.org/?distance=normal&duration=60&interaction=oneTime&personCount=2&riskProfile=average&scenarioName=indoorUnmasked2&setting=indoor&subLocation=US_06075&theirMask=none&theirVaccine=unvaccinated&topLocation=US_06&voice=normal&yourMask=none&yourVaccineDoses=2&yourVaccineType=pfizer

**To Reproduce**
See above link

**Is this a regression?**
Yes, seems recent

**Comment by sameerjain0123** [comment]
Possibly related: when running an estimate for San Francisco, I currently see a yellow box saying "Since our system doesn't yet have vaccine data for your region, we aren't able to provide a way for you to adjust your score based on whether or not the person you're seeing is vaccinated" and a risk profile that reflects that.

I was _not_ experiencing this issue yesterday (12/31/21, PST)

**Comment by jackc271** [comment]
Yes, that is also the behavior I see today (which is better than the incorrect vaccination rate data at least).
